pack.yaml

```diff
@@ -17,7 +17,8 @@ keywords:
   - CloudFormation
   - RDS
   - SQS
+  - lambda
 
-version : 0.7.7
+version : 0.8.0
```

Comment:
Current version of master branch on the remote repository is 0.7.9. It means your local repository is old. So you should merge current master branch to resolve this conflict.